1.0/docs/start/psk/setting-up.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Setting up
+subtitle: Polymer Starter Kit
+---
+
```

Comment:
:heavy_check_mark: added ToC macro to set-up and create-a-page. Not adding to deploy as it actually does not need one. 
:x: start each article with intro text